src/scrapers/platformen/scraper.py

```diff
@@ -102,14 +102,15 @@ class Scraper(ABC):
     Concept for a base class that defines and deals basic setup of a scraper 
     """
 
-    def __init__(self, platform_url: str, name: str, code: str, sources: List[PlatformSource] = []):
+    def __init__(self, platform_url: str, name: str, code: str, sources: List[PlatformSource] = None):
         # Leave out until full conversion of scrapers.
         # if len(sources) == 0:
         #    raise ValueError("Expecting at least one source!")
+        self._group = None
```

Comment:
Ik zou waarden die je niet expliciet in de `__init__` zet of gebruikt, gewoon als class variabelen opnemen; dan blijft je init wat leger